src/serviceFacades/users.js

```diff
@@ -130,10 +137,15 @@ function useSavePreferences(onSuccessCallback, onError) {
         onSuccess: (updatedPref) => {
             //update preference in cache
             queryCache.setQueryData(BOOTSTRAP_KEY, (bootstrapData) => {
-                if (bootstrapData && updatedPref) {
+                if (
+                    bootstrapData &&
+                    updatedPref &&
+                    updatedPref[0].preferences
+                ) {
+                    const updatePref = updatedPref[0].preferences;
```

Comment:
The response is in an array now cos we do a `Promise.all` 🤔 . Not sure about this...